internal/params/config.go

```diff
@@ -137,6 +146,12 @@ func (c *ChainConfig) IsCrossTx(epoch *big.Int) bool {
 	return isForked(crossTxEpoch, epoch)
 }
 
+// IsStaking determines whether it is staking epoch
+func (c *ChainConfig) IsStaking(epoch *big.Int) bool {
+	stkEpoch := new(big.Int).Add(c.StakingEpoch, common.Big1)
```

Comment:
no need to add 1. see the reasoning for IsCrossTx.

Reply:
removed